### DISCUSSION TITLE
How to first login

### DISCUSSION THREAD

**Discussion by levko-art** [vsdudakov/fastadmin#38] [Q&A] [answered]
Hello! How I can get credentials for my first login (I am use FastAPI + Tortoise ORM)

**Comment by akorsunov23** [comment, marked as answer]
> Hello! How I can get credentials for my first login (I am use FastAPI + Tortoise ORM)

Hello! It is necessary to create in the User model after "is_superuser" and add a user with the True value, he will be the administrator.

**Comment by vsdudakov** [comment]
Close the discussion
